### ISSUE TITLE
Download link

### ISSUE THREAD

**Issue by tavianator** [darchr/parsec-benchmark#4]
The PARSEC website http://parsec.cs.princeton.edu/ seems to be down.  Do you have an unmodified copy of `parsec-3.0.tar.gz` available?  Thanks!

**Comment by powerjg** [comment]
No, but if you go back to the initial commit, that will be very close.

**Comment by mahyarsamani** [comment]
If you checkout this repo at fc3e74e63ddf5c3cf658af190a9ebe7ac7f9dd18 it should be the version I started with.

**Comment by tavianator** [comment]
I need the datasets too though, I assume those are not in the repo?

**Comment by mahyarsamani** [comment]
I don't think so.

**Comment by voltavidTony** [comment]
Don't know if anyone is still looking after all this time, but another user uploaded his backups of the input files here https://github.com/cirosantilli/parsec-benchmark and has updated 'get-inputs' to download from his repo instead of the retired princeton website
